lib/graphql/schema/default_renamer.rb

```diff
@@ -0,0 +1,13 @@
+module GraphQL
+  class Schema
+    module DefaultRenamer
```

Comment:
This would be part of schema definition, not query execution, so personally, I'm not too concerned about it. If it becomes a problem, we can always add complexity in order to gain performance later. Or, if you'd like to check it out now, I could be convinced to prioritize performance _now_ instead of later by seeing a benchmark!